dashboards/grafana-dashboard-insights-rosocp-general.configmap.yaml

```diff
@@ -86,7 +86,7 @@ data:
                 "uid": "${datasource}"
               },
               "editorMode": "code",
-              "expr": "sum(kafka_consumergroup_group_lag{topic=~\".*hccm.ros.events\", group=\"ros-ocp\"})",
+              "expr": "sum(aws_kafka_sum_offset_lag_sum{topic=~\".*hccm.ros.events\", group=\"ros-ocp\"})",
```

Comment:
Is `group` valid label for this metric `aws_kafka_sum_offset_lag_sum` ?

Reply:
I believe this should be `consumer_group`, I'm have shared the reference link on DM.
Change pushed.